__tests__/app.test.js

```diff
@@ -109,4 +109,40 @@ describe("API Articles", () => {
         })
     });
   });
+  describe("GET /api/articles", () => {
+    it("200 Success - should return articles with the correct keys and a sum of each articles comments", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          response.body.articles.forEach((story) => {
```

Comment:
don't forget to make sure that any arrays you're iterating over in a test aren't empty (ie. checking the length), otherwise an empty array will give you a false positive on this test